blocks/HARDWARE/PROTOCOLS/SERIAL/OPEN_SERIAL/OPEN_SERIAL.py

```diff
@@ -29,4 +34,7 @@ def OPEN_SERIAL(device: SerialDevice, baudrate: int = 9600) -> TextBlob:
 
     DeviceConnectionManager.register_connection(device, ser)
 
+    if connection_time > 0:
+        sleep(connection_time)
```

Comment:
What's this for?

Reply:
Oh ok, I think this is fine, it makes it more obvious to the user that waiting could be required